### PR TITLE
Move success view to new route

### PR DESCRIPTION
**Pull request by RamRamez**
#3363

### CODE REVIEW

File: pages/success/[slug].tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+			props: {
+				project: data.projectBySlug,
+			},
+		};
+	} catch (error) {
+		return {
+			props: {},
+		};
+	}
+}
```
Comment (MohammadPCh) [inline pages/success/[slug].tsx:41]: @RamRamez Thanks, we don't need to build this page server side.

**Review by MohammadPCh — CHANGES_REQUESTED** [review]

File: pages/success/[slug].tsx (hunk 1 above)
Comment (RamRamez) [reply pages/success/[slug].tsx:41]: @MohammadPCh Do you mean removing getServerSideProps and fetching PROJECT_BY_SLUG client side?

Comment (MohammadPCh) [reply pages/success/[slug].tsx:41]: Yes, Thank you @RamRamez

Comment (RamRamez) [reply pages/success/[slug].tsx]: Done!

**Review by MohammadPCh — APPROVED** [review]
LGTM Thanks @RamRamez

**Pull request merged**